packages/common/src/themes/codesandbox-black.js

```diff
@@ -208,7 +208,7 @@ const colors = {
     background: tokens.grays[700],
     hoverBackground: tokens.green,
     border: tokens.grays[500],
-    foreground: tokens.white,
+    foreground: tokens.grays[300],
```

Comment:
Should this be `white` again?

Reply:
I changed these values, because of your changes in #3008